src/tests/topp/ProteomicsLFQTestScripts/run_lfq.sh

```diff
@@ -0,0 +1,51 @@
+#!bin/bash
+export PATH="$PATH:/home/sachsenb/OpenMS/openms-build-tmp/bin"
```

Comment:
we probably need to turn that into an integration test at some point.
should we remove those scripts for now?